app/javascript/packs/acuant_sdk_image_capture.js

```diff
@@ -0,0 +1,76 @@
+const acuantFallbackImageForm = () => document.querySelector('.acuant-fallback-image-form');
```

Comment:
this is a neat pattern, I like it how the DOM references don't get stale

Reply:
Yeah, I hate when my JS is littered with DOM references. Took it a little further in the next commit: https://github.com/18F/identity-idp/pull/3696/commits/1165578976929074038e8f6f2d05dc5a581249e4#diff-af03494139dfa52a5dba3065e3caae64R1

Reply:
Yes! Except that one seems to eagerly load the DOM elements? Instead of lazy-loading them in functions like this one?

Reply:
Yep, I just pushed a new commit that puts things back the way they were. The stale references were causing problems with the tests.